src/main/java/uk/gov/justice/hmpps/prison/aop/AuthorisationAspect.kt

```diff
@@ -114,8 +114,8 @@ class AuthorisationAspect(
     val method = signature.method
     val annotation = method.getAnnotation(VerifyStaffAccess::class.java)
     val overrideRoles = annotation.overrideRoles
-    if (!AuthenticationFacade.hasRoles(*overrideRoles)) {
-      val currentUsername: String = authenticationFacade.currentPrincipal
+    if (!HmppsAuthenticationHolder.hasRoles(*overrideRoles)) {
+      val currentUsername: String = authenticationFacade.username
```

Comment:
can be `username` here since we check immediately afterwards in the repository that the user exists